.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Bug Report
+about: Report Bugs
+title: ''
+labels: 'type: bug'
+assignees: ''
+
+---
+
+**Problems**
+
+**Screenshots/Demo**
```

Comment:
I strongly suggest we also add sections as shown in this [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#markdown-issue-template). It'll guide the reporter to write down as much as detail as possible 😃